packages/eslint-plugin-mobx/src/missing-make-observable.js

```diff
@@ -28,31 +53,67 @@ function create(context) {
         }
       } else {
         const fix = fixer => {
+          // The class extends a another unknown class so we can not safely create a super call.
```

Comment:
Doesn't matter, create `constructor` without `super` and let another eslint rule handle missing super.
I think that handling React component specifically as part of this rule is unnecessary as well, but I am ok with it.